filebeat/tests/system/test_harvester.py

```diff
@@ -421,7 +435,7 @@ def test_close_timeout(self):
         assert len(data) == 1
 
         # Check that not all but some lines were read. It can happen sometimes that filebeat finishes reading ...
-        assert self.output_lines() <= 1000
+        assert self.output_lines() <= 50
```

Comment:
This was set to 1000 to make sure in most cases 1000 is not reached. With a value of only 50 I think the likelyhood that all lines are read is very high which is not the intention of this test.

Reply:
Lets sync on that, I think the test was not exactly what you want before.

Reply:
I've talked to @ruflin I will rework that.